src/Apps/Order/Routes/__tests__/Shipping.jest.tsx

```diff
@@ -179,27 +179,82 @@ const saveAndContinue = async () => {
   await flushPromiseQueue()
 }
 
+const recommendedAddress = {
+  addressLine1: "401 Broadway Suite 25",
+  addressLine2: null,
+  city: "New York",
+  region: "NY",
+  postalCode: "10013",
+  country: "US",
+}
+
+const verifyAddressWithSuggestions = async (relayEnv, input, suggested) => {
```

Comment:
There was discussion about rspec-like shared example/context on Slack. I started by this basic helper only for DRYing the address verification step. The step can be plugged in (or removed from) the flow as the [example](https://github.com/artsy/force/pull/12939/files#diff-24b4bdbd70487ae4bc0352d30cf15dad7d22c4ea225bc9dec6ebd9eb8b23de99R1208-R1212) below.

Reply:
I'm in favor of trying this out, let's just be intentional about seeing it through.

non-blocking: As a matter of convention, since both this and `fillAddressForm` include expectations, would it be worth making clear that there is expectation behavior attached? We could just change the name, or we could look into [custom matchers.](https://jestjs.io/blog/2018/05/29/jest-23-blazing-fast-delightful-testing#custom-asynchronous-matchers)